### PR TITLE
MBS-12137: Remove Gravatar

### PR DESCRIPTION
**Pull request by mwiencek**
# MBS-12137

Related to MBS-9725.  Gravatar was scraped and email address hashes were cracked.  We should migrate away from their service.

I left in support for avatars in the codebase to minimize the amount of work required to add in support for Discourse avatars in the future.

### CODE REVIEW

File: lib/MusicBrainz/Server/Entity/Editor.pm [hunk 1]
```diff
@@ -275,15 +274,10 @@ sub new_privileged {
     );
 }
 
-sub gravatar {
+sub avatar {
```
Comment (reosarevok) [inline lib/MusicBrainz/Server/Entity/Editor.pm:277]: Maybe add a # TODO comment here?

**Review by reosarevok — APPROVED** [review]

File: lib/MusicBrainz/Server/Entity/Editor.pm (hunk 1 above)
Comment (yvanzo) [reply lib/MusicBrainz/Server/Entity/Editor.pm:280]: The ticket for “returning avatars from Discourse” is MBS-9725.
You might want to create a separate ticket to refer to the removal of Gravatar in this pull request.

**Comment by reosarevok** [comment]
@brainzbot, retest this please

File: lib/MusicBrainz/Server/Entity/Editor.pm (hunk 1 above)
Comment (mwiencek) [reply lib/MusicBrainz/Server/Entity/Editor.pm:280]: (Thanks for opening the ticket, I meant to do that sooner.)

**Review by yvanzo — APPROVED** [review]
The commit message should probably start with ticket reference `MBS-12137`

**Pull request merged**